docs/i18n/ja/docusaurus-plugin-content-docs/current/architecture-overview.md

```
@@ -0,0 +1,228 @@
+| `google-cloud`    | TypeScript | Cloud Firestore、Cloud KMS、Secret Manager など GCP サービスとの連携を提供    |
+
+
+## Reference Applications
+
+| パッケージ  | 言語    | 役割  |
+| --- | --- | --- |
+| `server/core`  | TypeScript | サンプルサーバーで共通利用するフレームワークおよび共通コンポーネントを提供   |
+| `server/single` | TypeScript | シングルテナント構成のサンプルサーバー   |
+| `server/multi`  | TypeScript | マルチテナント構成のサンプルサーバー   |
+| `server/aws`    | TypeScript | サンプルサーバーを AWS（Lambda + CDK）へデプロイするための構成例  |
+| `server/google-cloud` | TypeScript | サンプルサーバーを Google Cloud へデプロイするための構成例  |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)